emcc.py

```diff
@@ -3538,6 +3538,9 @@ def generate_html(target, options, js_target, target_basename,
                   memfile, optimizer):
   logger.debug('generating HTML')
 
+  if shared.Settings.EXPORT_NAME != 'Module':
+    exit_with_error('Customizing EXPORT_NAME is not currently possible with HTML output (see https://github.com/emscripten-core/emscripten/issues/10086)')
+
```

Comment:
Can this be dropped to non-MINIMAL_RUNTIME error only? In MINIMAL_RUNTIME the `EXPORT_NAME` field is utilized (https://github.com/emscripten-core/emscripten/blob/34ea2d91e87ee1d1fa589e08013cfcb2f59f030c/src/shell_minimal_runtime.html#L21)

Reply:
Thanks @juj, I didn't know that about the minimal runtime! Changed to only do this in non-minimal.